Volume/Supporting Views/PDFKitView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  PDFKitView.swift
+//  Volume
+//
+//  Created by Jennifer Gu on 11/12/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import PDFKit
+
+struct PDFKitView: UIViewRepresentable {
+    
+    let pdfDoc : PDFDocument
+    @Binding var showToolbars: Bool
+    
+    func makeCoordinator() -> Coordinator {
+        Coordinator(self)
+    }
+    
+    func makeUIView(context: Context) -> PDFView {
+        let pdfView = PDFView()
+        pdfView.document = pdfDoc
+        pdfView.autoScales = true
+        return pdfView
+    }
+    
+    func updateUIView(_ pdfView: PDFView, context: Context) {
+        pdfView.document = pdfDoc
+    }
+}
+
+struct SimplePDFView: UIViewRepresentable {
+    
+    let pdfDoc : PDFDocument
+    
+    func makeUIView(context: Context) -> PDFView {
+        let pdfView = PDFView()
+        pdfView.document = pdfDoc
+        pdfView.autoScales = true
+        return pdfView
+    }
+    
+    func updateUIView(_ pdfView: PDFView, context: Context) {
+        pdfView.document = pdfDoc
+    }
+}
```

Comment:
This is pretty much an exact copy of the PDFKitView declared above, could you find a way to reuse the code in them to suit both use cases? Could potentially just use PDFKitView by itself directly in MagazineCell.